storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operations/execute/PreparedExecuteSQL.java

```diff
@@ -113,7 +118,22 @@ public Object executeAsBlocking() {
     @CheckResult
     @Override
     public Observable<Object> asRxObservable() {
-        return RxJavaUtils.createObservable(storIOSQLite, this);
+        throwExceptionIfRxJavaIsNotAvailable("asRxObservable()");
+
+        final Set<String> observesTables = rawQuery.observesTables();
+        final Set<?> observesTags = rawQuery.observesTags();
+
+        final Observable<Object> observable;
+        if (!observesTables.isEmpty() || !observesTags.isEmpty()) {
+            observable = ChangesFilter.applyForTablesAndTags(storIOSQLite.observeChanges(), observesTables, observesTags)
```

Comment:
I didn't find it for [asRxObservable](https://github.com/pushtorefresh/storio/blob/master/storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operations/execute/PreparedExecuteSQL.java#L68), for [RawQuery](https://github.com/pushtorefresh/storio/blob/master/storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/queries/RawQuery.java#L270) and in the [doc](https://github.com/pushtorefresh/storio/blob/master/docs/StorIOSQLite.md#4-execsql-operation) too=)